app/models/help.rb

```diff
@@ -0,0 +1,6 @@
+class Help < ActiveRecord::Base
+  validates :name, length: { minimum: 2, maximum: 30 }
+  validates :messages, length: { maximum: 400 }
+  VALID_EMAIL_REGEX =  /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, format: { with: VALID_EMAIL_REGEX }
```

Comment:
There is no need to validate email, because it should be created automatically from `current_user`, which was already validated during registration